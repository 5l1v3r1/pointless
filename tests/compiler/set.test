
y = {"", 2, false}