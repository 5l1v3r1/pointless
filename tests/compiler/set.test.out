   0 [ SaveImports  ] 0       
   1 [ MakeEnv      ] 1       
   2 [ MakeNumber   ] 6       
   3 [ MakeDef      ] 0       (y)
   4 [ Jump         ] 7       (-> 11)
   5 [ LoadStr      ] 8       ("")
   6 [ MakeNumber   ] 2       
   7 [ LoadBool     ] 0       
   8 [ MakeSet      ] 3       
   9 [ SaveVal      ] 0       (y)
  10 [ Return       ]
  11 [ MakeObj      ]
  12 [ FieldRef     ] 9       (output)
  13 [ Jump         ] 6       (-> 19)
  14 [ Dup          ]
  15 [ LangFieldRef ] 16      (!getHead)
  16 [ Pop          ] 1       
  17 [ Jump         ] 2       (-> 19)
  18 [ LangFieldRef ] 25      (!getTail)
  19 [ Dispatch     ]
